### PR TITLE
Login test user to enforce storage creation

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description

Test requires change in order to let files_primary_s3 test run properly - see https://github.com/owncloud/files_primary_s3/pull/75

Failing acceptance tests can be ignore at the moment.

## How Has This Been Tested?
- unit tests execution in here 
- unit tests execution in https://github.com/owncloud/files_primary_s3/pull/75

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/lib/Encryption/DecryptAllTest.php [hunk 1]
```diff
@@ -524,6 +521,9 @@ public function testDecryptFile() {
 	 * Test to verify the fileid after the decryptFile method should be same
 	 */
 	public function testDecryptFileForFileId() {
+		if ($this->runsWithPrimaryObjectstorage()) {
+			$this->markTestSkipped('Objectstore cannot yet ensure to keep the file-id when copying/renaming files files');
```
Comment (phil-davis) [inline tests/lib/Encryption/DecryptAllTest.php]: ``files`` is duplicated.

**Comment by patrickjahns** [comment]
Needed to install core before installing objectstore app to be able to execute commands

Not sure if testing app modifies any behavior in core for unit testing - can you confirm @phil-davis @DeepDiver1975 @PVince81

**Comment by PVince81** [comment]
As far as I remember the testing app only modifies things for acceptance tests, not unit tests

**Comment by patrickjahns** [comment]
increased coverage 💪

**Review by patrickjahns — APPROVED** [review]
LGTM

**Comment by patrickjahns** [comment]
backport of drone changes required to always test `files_primary_S3` on core

**Comment by phil-davis** [comment]
@DeepDiver1975 another backport waiting to be done. I guess the backport would be good to get in the nightly image build.

**Pull request merged**